tests/Pinta.Effects.Tests/Mocks/MockPalette.cs

```diff
@@ -0,0 +1,18 @@
+using Cairo;
+using Pinta.Core;
+
+namespace Pinta.Effects.Tests;
+
+internal sealed class MockPalette : IPaletteService
+{
+	public Color PrimaryColor { get; set; } = new (0, 0, 0); // Black
+	public Color SecondaryColor { get; set; } = new (255, 255, 255); // White
```

Comment:
I think this is incorrect since `Cairo.Color` is floating point, so the 255's should be just 1 (look at the `ToColorBgra` method)
You could also just do `ColorBgra.White.ToCairoColor()`

Reply:
You are right. For a moment I forgot that in Cairo the color channels go from 0 to 1. It's corrected now